lib/errors/bulk-record-error.ts

```diff
@@ -10,12 +9,15 @@ const BaseError = require('./base-error');
  * @param {object} record  DAO instance that error belongs to
  */
 class BulkRecordError extends BaseError {
-  constructor(error, record) {
+  errors: Error;
```

Comment:
The parameter name here is misleading. It should be singular, it's just a small thing, but changing this could result in a breaking change.

@sdepold can we make a milestone for major releases (aka. v7), where we can reference this?

Reply:
Don't you have the permissions?

Reply:
We already have a milestone for v7: https://github.com/sequelize/sequelize/milestone/20

Reply:
Should we add a ticket to the v7 project?